src/app/application.cpp

```diff
@@ -278,7 +279,7 @@ void Application::torrentFinished(BitTorrent::TorrentHandle *const torrent)
         program.replace("%T", torrent->currentTracker());
         program.replace("%I", torrent->hash());
 
-        QProcess::startDetached(program);
+        std::thread ([program]() { return system(program.toStdString().c_str()); }).detach();
```

Comment:
>I thought we were discussing the impossibility to exit qbt until that command ends.

And I'm already planing how to workaround for the users.

how is that impossible to exit qbt when external program still running? what is the consequence? 
If my example is bad then draw a proper example and a bit explanation please.